### PR TITLE
#6 Added MIT License

### PR DESCRIPTION
**Pull request by nityanandagohain**
#6 I have added the LICENSE.md file.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Nityananda Gohain
```
Comment (DipanshKhandelwal) [inline LICENSE]: Change this to my name 'Dipansh Khandelwal'

**Comment by nityanandagohain** [comment]
done @DipanshKhandelwal

**Comment by DipanshKhandelwal** [comment]
@nityanandagohain  Please squash the commits.

**Comment by nityanandagohain** [comment]
I have squashed them @DipanshKhandelwal

**Comment by DipanshKhandelwal** [comment]
Closes #6

**Pull request merged**